bgpd/bgp_attr.h

```diff
@@ -251,7 +251,7 @@ struct attr {
 	as_t aggregator_as;
 
 	/* MP Nexthop length */
-	uint8_t mp_nexthop_len;
+	uint16_t mp_nexthop_len;
```

Comment:
Why is this needed? This can't be higher than 48 bytes overall (assuming it's encoded as global + link-local for vpnv6).

Reply:
your right I just assumed it was.  Sorry, removed